### PR TITLE
Fix issue #210

### PR DESCRIPTION
**Pull request by mmatera**
Just a couple of lines handles #210

<a href="https://gitpod.io/#https://github.com/Mathics3/mathics-core/pull/213"></a>

### CODE REVIEW

File: mathics/builtin/numbers/linalg.py [hunk 1]
```diff
@@ -194,21 +194,33 @@ class Inverse(Builtin):
...
      = Inverse[{{1, 0}, {0, 0}}]
 
-    >> Inverse[{{1, 0, 0}, {0, Sqrt[3]/2, 1/2}, {0,-1 / 2, Sqrt[3]/2}}]
-    = {{1, 0, 0}, {0, Sqrt[3] / 2, -1 / 2}, {0, 1 / 2, Sqrt[3] / 2}}
     """
 
-    messages = {"sing": "The matrix `1` is singular."}
+    messages = {
+        "sing": "The matrix `1` is singular.",
+        "matsq": ("Argument `1` at position 1 is not " "a non-empty square matrix."),
```
Comment (TiagoCavalcante) [inline mathics/builtin/numbers/linalg.py:201]: A bit weird style here (parenthesis and 2 strings).

**Review by TiagoCavalcante — APPROVED** [review]
LGTM

File: mathics/builtin/numbers/linalg.py (hunk 1 above)
Comment (mmatera) [reply mathics/builtin/numbers/linalg.py:201]: Sorry, it started being two lines...

Comment (mmatera) [reply mathics/builtin/numbers/linalg.py]: OK, it is clean now.

**Pull request merged**